classes/person.rb

```diff
@@ -27,4 +27,8 @@ def can_use_services?
   def correct_name
     @name
   end
+
+  def add_rental(rental)
+    rentals.push(rental)
+  end
 end
```

Comment:
- [ ] If you run this method, you will get an error that says `rentals` is not known. Using `rentals` would mean that there is a local scoped variable that has been declared. I would advise you to use the instance variable `@rentals` because it can be accessed by all methods in the class. Kindly use this after you have initialized the variable. 👍 😎 

```ruby
@rentals.push(rental)
```